### DISCUSSION TITLE
Sampler vs Scheduller on Flux Dev (Mac Studio Ultra M2)

### DISCUSSION THREAD

**Discussion by MigCurto** [lllyasviel/stable-diffusion-webui-forge#1803] [Show and tell]
I was getting very confused on what works or doesnt in Flux@Forge so decided to test all possible combinations (apart from the bottom 2 Align Your Steps) , might be of use for somebody else, its a 42MB JPG:

Flux Dev Sampler vs Scheduller JPG:
https://1drv.ms/i/s!AgCCOlE9ZwmSohqsgV89tODIX2pD?e=p7VaOB

10 Steps, CFG 1:
![Screenshot 2024-09-12 at 13 07 31](https://github.com/user-attachments/assets/44cf4816-00d9-4ea7-9a47-bd90e1de4804)

![image](https://github.com/user-attachments/assets/51cc2d73-c2b2-4b01-9981-cd4c4291dfa0)

**Comment by wisdombox44** [comment]
This may help
https://youtu.be/WtmKyqi_aFM?si=NTFlehsfp-BcWExO
